src/runtime.js

```diff
@@ -49,6 +50,17 @@ const TAG = 'runtime';
 /** @type {!Array} */
 const elementsForTesting = [];
 
+// TODO
```

Comment:
Please factor into function. Actually please factor into file `install-sw.js` and I think it should be called from `amp.js` instead